src/config.js

```diff
@@ -2,7 +2,7 @@
 
 const {parseStringToBoolean} = require('./util')
 
-exports.getConfig = function () {
+exports.getConfig = function() {
```

Comment:
Very minor: But i think we can keep the space here. This would be similair to writing something liek the below
`if () {`
compared to 
`if() {`

Reply:
This is also why travis is currently failing, on the lint error :)